fem/bilininteg_vectorfe.cpp

```diff
@@ -15,25 +15,18 @@
 namespace mfem
```

Comment:
Why would we split it? It is only about 1000 lines.

Reply:
To put the `Assemble` & `MultAdd` of `div` or `curl` together in some other file to avoid the forward of the instantiation, to allow parallelization of the compilation of these files. 
We could also uniformise the name with the other existing one.

Reply:
The H(div) kernels are in bilininteg_hdiv.cpp, and the H(curl) kernels are in bilininteg_hcurl.cpp, and this file only contains `VectorFEMassIntegrator` and `MixedVectorGradientIntegator` member functions which call those kernels in other files, except for a few mixed H(curl)-H(div) kernels. I don't see a better way to organize things in separate files.